src/Network/Send/Zero.pm

```diff
@@ -22,16 +22,18 @@ sub new {
 	my ($class) = @_;
 	my $self = $class->SUPER::new(@_);
 
-	my %packets = (		
-		'0ACF' => ['master_login', 'a4 Z25 a32 a5', [qw(game_code username password_rijndael flag)]],
+	my %packets = (
+		'0439' => ['item_use', 'a2 a4', [qw(ID targetID)]],
```

Comment:
packet item_use is 0439 ?

Reply:
need to test, but as someone sayd in discord can be 0439

Reply:
its working in 0439